packages/validator/test/cli-validator/mock-files/openapi-3-1.json

```diff
@@ -0,0 +1 @@
+{ "swagger": "2.0" }
```

Comment:
Shouldn't this one contain openapi 3.1?

Reply:
Haha, can't believe I didn't catch this!

Reply:
Hah. Oh man, when I duplicate a file, my editor sometimes confuses me on which on I'm in.

Reply:
haha, might be time to switch to a different editor :joy:
I hear that emacs is available LMAO